lib/avromatic/rspec.rb

```diff
@@ -0,0 +1,9 @@
+require 'webmock/rspec'
+require 'avro_turf/test/fake_schema_registry_server'
+
+RSpec.configure do |config|
+  config.before(:each) do
+    WebMock.stub_request(:any, /^#{Avromatic.registry_url}/).to_rack(FakeSchemaRegistryServer)
+    FakeSchemaRegistryServer.clear
+  end
+end
```

Comment:
There aren't currently any other test utilities but I'm inclined to make this `lib/avromatic/rspec/fake_registry_server.rb`